js/views/components/Moderators.js

```diff
@@ -115,55 +158,62 @@ export default class extends baseVw {
     const buyerCur = app.serverConfig.cryptoCurrency;
     const modCurs = profile.moderatorInfo && profile.moderatorInfo.acceptedCurrencies || [];
     const validCur = modCurs.includes(buyerCur);
-    // if the moderator is on the list of IDs to exclude, or is this user, remove them
-    const excluded = this.excludeIDs.includes(profile.peerID) || profile.peerID === app.profile.id;
```

Comment:
Handling excluded peerIDs was moved to the .done handler for getModeratorsByID, since it makes more sense not to process excluded IDs at all.